### PR TITLE
Private extension marketplace is currently experimental, not Beta

### PR DESCRIPTION
**Pull request by Frezzle**
### Proposed changes

We want to release extensions private marketplace as experimental, not beta. This PR corrects our communication.

Before:
Screenshot 2023-11-28 at 16 13 05

After:
Screenshot 2023-11-28 at 16 08 46

### Related issues (optional)

- [ ] Can I link internal JIRA issues here?

### CODE REVIEW

File: content/desktop/extensions/private-marketplace.md [hunk 1]
```diff
@@ -4,12 +4,12 @@ keywords: Docker Extensions, Docker Desktop, Linux, Mac, Windows, Marketplace, m
 title: Configure a private marketplace for extensions
 ---
 
-> **Beta**
+> **Warning**
```
Comment (aevesdocker) [inline content/desktop/extensions/private-marketplace.md:7]:
```suggestion
> **Experimental**
```

File: content/desktop/extensions/private-marketplace.md [hunk 2]
```diff
@@ -4,12 +4,12 @@ keywords: Docker Extensions, Docker Desktop, Linux, Mac, Windows, Marketplace, m
...
 >
-> Docker Extensions` private marketplace is in [beta](../../release-lifecycle#beta).
+> Docker Extensions` private marketplace is currently [experimental](../../release-lifecycle#experimental).
 >
 > It's available to Docker Business customers only.
-{ .experimental }
+{ .warning }
```
Comment (aevesdocker) [inline content/desktop/extensions/private-marketplace.md:12]:
```suggestion
{ .restricted }
```

**Review by aevesdocker — COMMENTED** [review]
Thank you @Frezzle, just a small tweak to the banner. Use of the warning banner should be similar to rules on the Design System - so keeping it for when an action can't be undone etc :)

**Review by aevesdocker — APPROVED** [review]

**Comment by Frezzle** [comment]
@aevesdocker Thanks Allie, looks way more inviting ✨ 

Screenshot 2023-11-28 at 16 23 22

**Pull request merged**